### DISCUSSION TITLE
Show file name in taskbar instead of path of file

### DISCUSSION THREAD

**Discussion by vuslatx** [d2phap/ImageGlass#1276] [General] [answered]
Is there any settings about showing file name in taskbar instead of path of file? If not, it would be better if it be.

**Comment by d2phap** [comment, marked as answer]
Yes, this:

![image](https://user-images.githubusercontent.com/3154213/151312520-a6634e0b-2c69-42f6-9dc0-4bb01f1d4d99.png)
